examples/hosted-wallet-kt/wallet/src/main/kotlin/ExampleCancelFlow.kt

```diff
@@ -0,0 +1,93 @@
+import tbdex.sdk.messages.*
+import tbdex.sdk.web5.BearerDid
+
+private lateinit var webhook: ReplyToWebhook
+private var closedReceived = false
+
+fun runCancelFlow(
+    pfiDidUri: String,
+    verifiableCredential: String,
+    bearerDid: BearerDid,
+    replyToUrl: String
+) {
+    println("\n ~Running Cancel Flow~ \n")
```

Comment:
the ~ makes me think of 

![image](https://github.com/user-attachments/assets/8b66d9a9-1dc0-41f0-8222-f3f6bb1d0e4b)